.bazelrc

```diff
@@ -40,10 +40,13 @@ build --enable_platform_specific_config
 build --action_env=CC --host_action_env=CC
 build --action_env=CXX --host_action_env=CXX
 build --action_env=LLVM_CONFIG --host_action_env=LLVM_CONFIG
-# Do not pass through PATH however.
-# It tends to have machine-specific values, such as dynamically created temp folders.
-# This would make it impossible to share remote action cache hits among machines.
-# build --action_env=PATH --host_action_env=PATH
+# Do not pass machine-specific values through PATH however.
```

Comment:
This file is kept in sync with the Envoy's version.

Let's only modify the lines we have to modify. It is ok to add additional comment. Let's also mark all lines we are modifying as unique.

Reply:
Updated comment.  Now it is in sync with Envoy's version. Added lines with `#unique` marks.